designer/apps/pages/models.py

```diff
@@ -196,16 +196,22 @@ class ProgramPageBranding(Branding):
 
 class EnterprisePage(Page):
     """
-    Used to store information for a program on a site
+    Used to store information for an Enterprise on a site
 
     .. no_pii:
     """
     uuid = models.UUIDField(unique=True)
+    contact_email = models.EmailField(
+        blank=True,
```

Comment:
Note: Not sure if this field should be required. I'm guessing not, so I have `blank=True` here. The frontend code handles when `contact_email` is empty gracefully 😃